src/app/Components/Authorization/sign-up/sign-up.component.html

```diff
@@ -1,15 +1,11 @@
 <div id="GlobalDiv" fxLayout="row" fxLayoutAlign="center center">
   <mat-card>
-
     <mat-card-header>
       <mat-card-title>Registration</mat-card-title>
     </mat-card-header>
-
-    <form fxLayout="row" fxFlexAlign="space-evenly center" [formGroup]="service.form" 
-    #signUpForm="ngForm" class="normal-form" (ngSubmit)="onSubmit(signUpForm)" enctype="multipart/form-data">
-
+    <form fxLayout="row" fxFlexAlign="space-evenly center" [formGroup]="service.form" #signUpForm="ngForm" class="normal-form"
```

Comment:
#signUpForm="ngForm" must be removed and "onSubmit(signUpForm") changed to "onSubmit()".